tests/Util/InstantiatorTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\AdminBundle\Exception\AbstractClassException;
+use Sonata\AdminBundle\Tests\Fixtures\Entity\AbstractEntity;
+use Sonata\AdminBundle\Util\Instantiator;
+
+/**
+ * @author Morgan Abraham <morgan@geekimo.me>
+ */
+final class InstantiatorTest extends TestCase
```

Comment:
It would be great to have another test that checks if a non-abstract class can be instanciated.

Reply:
Done, I've added another case.